controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/application/JobControllerApiHandlerHelper.java

```diff
@@ -395,6 +391,12 @@ static HttpResponse overviewResponse(Controller controller, TenantAndApplication
                     toSlime(runObject.setObject("versions"), versions.versions(), application);
                 }
 
+                if ( ! jobStatus.runs().isEmpty())
+                    controller.applications().decideCloudAccountOf(new DeploymentId(job.application(),
```

Comment:
I was hoping we could make `decideCloudAccountOf()` private and always use `Deployment::cloudAccount()`

Reply:
Doesn't work for test jobs which have only temporary deployments, though.

Reply:
We'd have to add it to each run in that case.